r/R/feather.R

```diff
@@ -15,13 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
-#' Write data in the Feather format
+#' Write a Feather file (an Arrow IPC file)
 #'
 #' Feather provides binary columnar serialization for data frames.
 #' It is designed to make reading and writing data frames efficient,
 #' and to make sharing data across data analysis languages easy.
-#' This function writes both the original, limited specification of the format
-#' and the version 2 specification, which is the Apache Arrow IPC file format.
+#' [write_feather()] can write both the Feather Version 1,
+#' a legacy version available starting in 2016, and the Version 2,
+#' which is the Apache Arrow IPC file format.
+#' [write_ipc_file()] is a shortcut to `write_feather(version = 2)`.
```

Comment:
I think we should note that version 2 is default, so `write_ipc_file(df, filename)` does the same as `write_feather(df, filename)`